Code/src/main/java/nl/utwente/group10/ui/menu/CircleMenu.java

```diff
@@ -31,25 +31,29 @@ public CircleMenu(Block block) {
         this.block = block;
 
         // Define menu items
-
+        
         // Cut Option
-        MenuItem delete = new MenuItem("cut", new ImageView(new Image(this
-                .getClass().getResourceAsStream("/ui/cut32.png"))));
+        ImageView image = new ImageView(new Image(this.getClass().getResourceAsStream("/ui/cut32.png")));
+        image.getStyleClass().add("hoverMenu");
+        MenuItem delete = new MenuItem("cut", image);
```

Comment:
Could refactor this into a `private static ImageView makeImageView(String path)` factory method.